### DISCUSSION TITLE
夸克TV速度慢

### DISCUSSION THREAD

**Discussion by Simonzhang8** [AlistGo/alist#7310] [General]
用夸克TV+302的模式播放视频，速度还是特别慢，测试下载只有80k，有什么好的解决方法吗？
（已经开了SVIP）

**Comment by 1109wuwuwu** [comment]
我也是，楼主解决了吗？

**Comment by git-liweichao** [comment]
请问有解决的方法吗？

**Comment by lilu0826** [comment]
我也是，感觉被限速了，只有100KB/S不到。

**Comment by verydemo** [comment]
@xfoxfu 大神， 夸克有这个问题

**Comment by SeanDragon** [comment]
@xfoxfu 呼叫呼叫，大佬

**Comment by dgscyg** [comment]
> @xfoxfu 呼叫呼叫，大佬

夸克 TV 限速规则

`线程数 * 400MB > 文件大小 `时不限速

播放一般都是单线程，所以只有播放 400MB以下的视频才不限速。

**Reply by hansostong** [reply]
刚测试了下，也会限速
